frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.ts

```diff
@@ -127,6 +135,29 @@ export const filtersToQueryNode = (filters: Partial<FilterType>): InsightQueryNo
 
     // breakdown
     if (isInsightQueryWithBreakdown(query)) {
+        // handle multi-breakdowns
+        if (filters.breakdowns != null) {
+            if (filters.breakdowns.length === 1) {
+                filters.breakdown_type = filters.breakdowns[0].type
+                filters.breakdown = filters.breakdowns[0].property as string
+            } else {
+                captureException(
+                    'Could not convert multi-breakdown property `breakdowns` - found more than one breakdown'
+                )
+            }
+        }
+
+        // handle missing breakdown_type
+        if (filters.breakdown != null && filters.breakdown_type == null) {
+            if (Array.isArray(filters.breakdown)) {
+                filters.breakdown_type = 'cohort'
+            } else if ((filters.breakdown as string).startsWith('$initial_')) {
+                filters.breakdown_type = 'person'
```

Comment:
This feels kind of magical. Do we really have some backend logic that chooses the type if the value starts with `$initial_`? 🤔 

If we can get away without this `if` clause, we should, but if not, then 🤷

Reply:
Backend falls back to "event" type if it's missing, but I think this is actually wrong here see e.g. this insight for our team: https://app.posthog.com/insights/gRyK5yhP/edit.

I'll still remove the if clause, as nobody is complaining that the insights are wrong right now :)